.github/workflows/ci_backend.yml

```diff
@@ -6,34 +6,39 @@ on:
       - master
       - dev
       - 'releases/**'
-    paths:
-      - 'RetrospectiveExtension.Backend/**'
   pull_request:
     branches:
       - master
       - dev
       - 'releases/**'
-    paths:
-      - 'RetrospectiveExtension.Backend/**'
   workflow_dispatch:
 
 jobs:
   build:
-
-    strategy:
-      fail-fast: false
-
+    name: Backend - Build and Test
     runs-on: ubuntu-latest
-
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
-
-    - name: Initialize CodeQL
-      uses: github/codeql-action/init@v1
-
-    - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
-
-    - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v1
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            src:
+              - '.github/**'
+              - 'RetrospectiveExtension.Backend/**'
+              - 'RetrospectiveExtension.Backend.Tests/**'
+
+      - if: steps.changes.outputs.src == 'true'
```

Comment:
Is there a way to check the false state and abort? Or combine the steps under a single if? This could be a future enhancement.

Reply:
Doesn't look like GitHub Actions support it. Will spend more time investigating in the future.